product-service-ts/serverless.ts

```diff
@@ -47,6 +48,30 @@ const serverlessConfiguration: Serverless = {
           http: {
             method: 'get',
             path: 'products/{id}',
+            cors: true,
+          }
+        }
+      ]
+    },
+    createProduct: {
+      handler: 'handlers/createProduct.handler',
+      events: [
+        {
+          http: {
+            method: 'put',
```

Comment:
By task you need to use POST for createProduct handler
![image](https://user-images.githubusercontent.com/17451948/98723264-30c83280-23a3-11eb-9fa0-4ee0e225a6d4.png)